src/sentry/utils/data_filters.py

```diff
@@ -0,0 +1,48 @@
+"""
+sentry.utils.data_filters.py
+~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import fnmatch
+
+
+class FilterTypes(object):
+    ERROR_MESSAGES = 'error_messages'
+    RELEASES = 'releases'
+    BLACKLISTED_IPS = 'blacklisted_ips'
+
+
+def is_valid_release(release, project):
```

Comment:
Good question (and observation), and I don't have any objective answer to it.

My personal opinion would be to change them all to include project as the first argument, but I can see how you could make a reasonable argument for not doing that too.

I was also going to mention that since it's in a different module it's less of a concern, but it only looks like it's actually used alongside these functions, which makes me wonder if it should really be in /this/ file instead? It doesn't really have anything to do with `http` anyway so its questionable whether or not it should actually live where it currently does